tracer/build/_build/Build.GitHub.cs

```diff
@@ -813,14 +813,17 @@ async Task HideCommentsInPullRequest(int prNumber, string prefix)
         return (artifactBuild, artifact);
     }
 
-    static async Task DownloadAzureArtifact(AbsolutePath outputDirectory, BuildArtifact artifact)
+    async Task DownloadAzureArtifact(AbsolutePath outputDirectory, BuildArtifact artifact)
```

Comment:
Can we keep this static, and pass `AzureDevopsToken` in as a parameter? That way we can more easily verify that any `Target` that _calls_ this method is marked with `Require(() => AzureDevopsToken)`
```suggestion
    static async Task DownloadAzureArtifact(AbsolutePath outputDirectory, BuildArtifact artifact, string azureDevopsToken)
```

Reply:
Sure, done in e69a14d0127637cfc083d16eb0c018d565eaffe1